README.md

```diff
@@ -33,7 +33,7 @@ https://github.com/askui/intent-pilot/assets/106730702/582d7dec-e3ff-43fd-9ab7-0
 
 - Python 3.9 or later
```

Comment:
We will have to update Python requirement to 3.11+
```
Traceback (most recent call last):
  File "/home/manu/anaconda3/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/manu/anaconda3/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/manu/Desktop/askui/intent-pilot/src/intent_pilot/__main__.py", line 2, in <module>
    from intent_pilot.run.app import main
  File "/home/manu/Desktop/askui/intent-pilot/src/intent_pilot/run/app.py", line 17, in <module>
    from enum import StrEnum
ImportError: cannot import name 'StrEnum' from 'enum' (/home/manu/anaconda3/lib/python3.10/enum.py)
```

Reply:
Nice catch. Updated accordingly.